cmd/ctr/commands/images/import.go

```diff
@@ -123,12 +123,18 @@ If foobar.tar contains an OCI ref named "latest" and anonymous ref "sha256:deadb
 			return closeErr
 		}
 
+		dcparameters, err := CreateDcParameters(context, nil)
+		if err != nil {
+			return err
+		}
+
 		if !context.Bool("no-unpack") {
 			log.G(ctx).Debugf("unpacking %d images", len(imgs))
 
 			for _, img := range imgs {
 				// TODO: Allow configuration of the platform
 				image := containerd.NewImage(client, img)
+				image.SetDecryptionParameters(dcparameters)
```

Comment:
This defiantly does not belong on an Image.

Reply:
> ```go
> client.Pull(ctx, ref, containerd.WithImageUnpack)
> ```

You meant the existing `containerd.WithPullUnpack` ?

Reply:
this looks like this now:
```
err = image.Unpack(ctx, context.String("snapshotter"), containerd.WithDcParameters(dcparameters))
```